clion/src/193/main/resources/META-INF/platform-clion-only.xml

```diff
@@ -1,2 +1,3 @@
-<idea-plugin>
+<idea-plugin xmlns:xi="http://www.w3.org/2001/XInclude" allow-bundled-update="true">
+    <xi:include href="/META-INF/coverage-only.xml" xpointer="xpointer(/idea-plugin/*)"/>
```

Comment:
Let's avoid such unexpected mixes of plugin xmls if possible
Looks like it's enough to change that `coverage-only.xml` depends on.
At this moment we use `Coverage` id to add an optional dependency on coverage plugin in root `plugin.xml` but if you convert it to module dependency (`com.intellij.modules.coverage` id), everything will work as expected
Also, don't forget to update the corresponding dependency in `plugin.xml` in `coverage/src/test/resources/META-INF`